packages/components-js/tests/e2e/puppeteer/helpers/puppeteer-helper.ts

```diff
@@ -1,7 +1,7 @@
 import type { ConsoleMessage, ElementHandle, Page, WaitForOptions, SnapshotOptions } from 'puppeteer';
 import { waitForComponentsReady } from './stencil';
 import type { TagName } from '@porsche-design-system/shared';
-import { ComponentMeta, getComponentMeta } from '@porsche-design-system/shared';
+import { ComponentMeta, getComponentMeta } from '@porsche-design-system/component-meta';
```

Comment:
unused import of `ComponentMeta`

```suggestion
import { getComponentMeta } from '@porsche-design-system/component-meta';
```